src/mapgen_fractal.cpp

```diff
@@ -49,17 +49,18 @@ FlagDesc flagdesc_mapgen_fractal[] = {
 MapgenFractal::MapgenFractal(int mapgenid, MapgenFractalParams *params, EmergeManager *emerge)
 	: MapgenBasic(mapgenid, params, emerge)
 {
-	this->spflags    = params->spflags;
-	this->cave_width = params->cave_width;
-	this->fractal    = params->fractal;
-	this->iterations = params->iterations;
-	this->scale      = params->scale;
-	this->offset     = params->offset;
-	this->slice_w    = params->slice_w;
-	this->julia_x    = params->julia_x;
-	this->julia_y    = params->julia_y;
-	this->julia_z    = params->julia_z;
-	this->julia_w    = params->julia_w;
+	this->spflags          = params->spflags;
```

Comment:
I tried removeng these 'this-> ..' lines and mapgen broke.
Maybe some of these are needed to distinguish private member variables of MapgenV7 from the same-name parameters in MapgenV7Params. But then i read that it's better to use a 'm_' prefix for the private member variables to distinguish.
I'll experiment and see, then cleanup all this in a commit.